gcp_variant_transforms/beam_io/vcfio_test.py

```diff
@@ -101,11 +101,14 @@ def _get_sample_variant_1(is_for_nucleus=False):
         vcfio.VariantCall(name='Sample2', genotype=[1, 0], info={'GQ': 20}))
   else:
     # 0.1 -> 0.25 float precision loss due to binary floating point conversion.
-    vcf_line = ('20	1234	rs123;rs2	C	A,T	50	'
+    # rs123;rs2 -> rs123 it seems nuclues does not parse IDs correctly.
+    # quality=50 -> 50.0 nucleus converts quality values to float.
+    # TODO(samanvp): convert all quality values to float.
```

Comment:
@allieychen FYI
I actually noticed that we have the same problem for the BQ -> VCF pipeline. The VCF quality values are typically integers, but the spec has float. So, while technically correct, our output VCF files have `#.0` for the quality field.
I think a better solution is to add some logic to our parser to output these as ints if the value is actually an integer (an easy way to do this is: `return quality if int(quality) != quality else int(quality)` (with some special casing when quality is None). Let's do this in another PR though.

Reply:
Got it! Thanks!